engine/src/main/java/org/terasology/engine/core/module/ModuleListDownloader.java

```diff
@@ -59,13 +59,13 @@ public ModuleRegistry call() throws IOException {
                 String json = gson.toJson(jObject);
 
                 ModuleMetadata meta = metaReader.read(new StringReader(json));
-                logger.debug("Read module {} - {}", meta.getId(), meta.getVersion());
+                logger.atDebug().addArgument(() -> meta.getId()).addArgument(() -> meta.getVersion()).log("Read module {} - {}");
```

Comment:
Simple getter methods so impact neglectable
```suggestion
                logger.debug("Read module {} - {}", meta.getId(), meta.getVersion()); //NOPMD
```